src/FlaUILibrary/keywords/property.py

```diff
@@ -64,24 +68,37 @@ def get_property_from_element(self, identifier, action, msg=None):
 
         """
         # pylint: enable=line-too-long
-
-        module = self._container.get_module()
-        element = module.get_element(identifier, msg=msg)
+        
         action_value = ""
-
         try:
             action_value = Property.Action[action.upper()]
-        except KeyError:
-            FlaUiError.raise_fla_ui_error(FlaUiError.InvalidPropertyArgument)
-
-        if action_value is (Property.Action.MAXIMIZE_WINDOW
-                            or Property.Action.MINIMIZE_WINDOW
-                            or action_value == Property.Action.NORMALIZE_WINDOW):
+        except KeyError or action_value in [Property.Action.MAXIMIZE_WINDOW, Property.Action.MINIMIZE_WINDOW, Property.Action.NORMALIZE_WINDOW]:
             FlaUiError.raise_fla_ui_error(FlaUiError.InvalidPropertyArgument)
+        
+        module = self._container.get_module()
+        
+        # need expand parent ComboBox before to get ComboBox SelectionItem element
+        is_combobox_selectionitem = True if "ComboBox" in identifier and "ComboBox" not in identifier.split("/")[-1] else False
```

Comment:
So that other actions are not affected, it would be better to query directly with an IF statement whether the selected action is used here. This is only valid for a small set of actions.

IF actions_value is Property.Action.IS_SELECTED

The check that this is a ComBox is then omitted because you expect this in your action as Xpath.

Reply:
You want only to collapse and extend by Action "IS_SELECTED "